game/entities.py

```
@@ -0,0 +1,252 @@
+            if x_end > x_start:  # Avoid division by zero
+                for x in range(x_start, x_end + 1):
+                    t = (x - x_start) / (x_end - x_start)
+                    self.heights[x] = int(y_start * (1 - t) + y_end * t)
+        
+        # Add some roughness to make it more interesting
+        self._add_roughness()
+        
+        # Create landing zones
+        self._create_landing_zones()
+    
+    def _add_roughness(self, roughness=0.15):
```

Ruff (0.8.2)

Missing return type annotation for private function `_add_roughness`

Add return type annotation: `None`

(ANN202)